lib/Steps/Integrations/NextJs.ts

```diff
@@ -176,25 +181,26 @@ export class NextJs extends BaseIntegration {
     }
   }
 
-  private _fulfillsMinVersion(proposed: string): boolean {
+  private _fulfillsMinVersion(version: string): boolean {
     // The latest version, which at the moment is greater than the minimum
     // version, shouldn't be a blocker in the wizard.
-    if (proposed === 'latest') {
+    if (version === 'latest') {
       return true;
     }
-    const [
-      proposedMajor,
-      proposedMinor = '0',
-      proposedPatch = '0',
-    ] = proposed.split('.');
-    const [minMajor, minMinor, minPatch] = MIN_NEXTJS_VERSION.split('.');
-    if (
-      parseInt(minMajor) > parseInt(proposedMajor) ||
-      parseInt(minMinor) > parseInt(proposedMinor) ||
-      parseInt(minPatch) > parseInt(proposedPatch)
-    ) {
+
+    const cleanedVersion = clean(version);
+    if (cleanedVersion) {
```

Comment:
When would `cleanedVersion` not be defined?

Reply:
Thought about adding these comments, but I think it's better to read the docs of the function/module. If the module gets updated, docs may get outdated and not represent what the code is doing. IMO, having comments stating the code to be doing a thing it actually isn't (even worse if there're examples) is worse than having to look at the documentation to know what it's doing. The comment I added for `gte` contradicts what I'm saying though. Wdyt?

Reply:
I am 100% Team Comment Your Code. We pin the version of the dependency, so things shouldn't change out from under us without us knowing, and as a reader of the code I def do not want to have to go read the docs for some other library, just to understand what our code is doing.

I missed your replies and I see that you just merged it, so NBD, but in the future, I think it is definitely better to answer whatever questions you anticipate the reader might have than hope they'll go read docs elsewhere.